src/components/ShortcutLink.jsx

```diff
@@ -1,24 +1,33 @@
-import React from 'react'
+import React, { useState } from 'react'
 import get from 'lodash/get'
+import { useOnLongPress } from 'rooks'
 
 import { useClient, useFetchShortcut, models } from 'cozy-client'
 import flag from 'cozy-flags'
+import ShortcutActionsMenu from './Shortcuts/ShortcutActionsMenu'
 
 const {
   file: { splitFilename, getShortcutImgSrc }
 } = models
 
 import SquareAppIcon from 'cozy-ui/transpiled/react/SquareAppIcon'
+import Badge from 'cozy-ui/transpiled/react/Badge'
 import Link from 'cozy-ui/transpiled/react/Link'
 
 export const ShortcutLink = ({ display = 'compact', file, ...props }) => {
   const client = useClient()
+
   const { shortcutInfos, shortcutImg, fetchStatus } = useFetchShortcut(
     client,
     file._id
   )
   const isLoading = fetchStatus === 'loading'
 
+  const [isMenuOpen, setIsMenuOpen] = useState(false)
+  const longPressRef = useOnLongPress(() => {
```

Comment:
we sure it's not necessary to add `isMobile` condition somewhere when using `useOnLongPress` to avoid weird behavior on desktop?

Reply:
Very good point. Fixed 👍